src/store/model/gamestop.ts

```diff
@@ -0,0 +1,19 @@
+import {Store} from './store';
+
+export const Gamestop: Store = {
+	labels: {
+		inStock: {
+			container: '#primary-details .primary-details-row .cart-and-ipay.divider-line.no-border-mobile',
```

Comment:
You may just want to use one container or class that is relevant.

```suggestion
			container: '.add-to-cart-buttons',
```

`.add-to-cart-buttons` seems like a good candidate. It's on other stocked items as well.

![image](https://user-images.githubusercontent.com/12074633/94631509-ae722c00-0296-11eb-80bc-9a23487aede0.png)

Reply:
yeah, I think just `.add-to-cart` works in newer testing. Still doesn't solve the issue with the outerHTML being forced on the store.lables.inStock test.

Reply:
This would use `textContent`, but not sure how to code with with all the strict typing things that are going on. Never used typescript before.

Reply:
No worries! Yeah in this case, you don't need to have the typings within this scope. We just put in the element.